mdaviz/utils.py

```diff
@@ -81,19 +80,19 @@ def get_det(mda_file_data):
     The first detector is labeled D01 and subsequent detectors follow in numerical order.
 
     Parameters:
-    - mda_file_data: An instance of an mda.scanDim object, which contains the MDA file data to be processed.
+        - mda_file_data: An instance of an mda.scanDim object, which contains the MDA file data to be processed.
 
     Returns:
-    - A tuple containing:
-      - A dictionary (d) where keys are indexes, mapping to either scanPositioner or scanDetector objects.
-        The dictionary is structured as {0: p0, 1: P1, ..., np: D01, np+1: D02, ..., np+nd: DX}.
-      - The index (first_pos) of the first positioner in the returned dictionary. This is 1 if a positioner
-        other than the default index positioner exists, otherwise 0.
-      - The index (first_det) of the first detector in the returned dictionary, which directly follows the last positioner.
-
-    Note:
-    - p0 is created by default and corresponds to the point index, described as an 'Index' scanPositioner object with predefined properties.
-    - np is the total number of positioners, nd the number of detectors, and npts the number of data points actually acquired.
+        - A tuple containing:
+            - A dictionary (d) where keys are indexes, mapping to either scanPositioner or scanDetector objects.
+                The dictionary is structured as {0: p0, 1: P1, ..., np: D01, np+1: D02, ..., np+nd: DX}.
+            - The index (first_pos) of the first positioner in the returned dictionary. This is 1 if a positioner
+                other than the default index positioner exists, otherwise 0.
+            - The index (first_det) of the first detector in the returned dictionary, which directly follows the last positioner.
+
+    ..note::
```

Comment:
The `..notes::` is a syntax error in the documentation language.  

Better to just write `Notes`.

Also, the indentation on the following lines will render that text as computer code.  Remove the indentation and should look better.

Reply:
If you want the word Notes to stand out, then replace `..notes::` with `.. rubric:: Notes`` (pay attention to the space after the two dots).

Reply:
I don't really care about using `..notes::`, I just used it because you suggested that at some point for another docstring (in the previous PR). Notes works just fine by me. I got to this:

![image](https://github.com/BCDA-APS/mdaviz/assets/39918069/355640ee-243c-4ebc-bd55-b4769fb0ce99)


I do not understand why the 2 first bullets in "returns" are in bold. I spent way too much time on it so I think this is minor enough that I just going to move on...

Reply:
Bold face in bullet points is unexpected.

![image](https://github.com/BCDA-APS/mdaviz/assets/2279984/2b35ad9b-e425-42ec-adaa-b66496cf9545)